src/main/java/org/jenkinsci/plugins/DependencyTrack/DependencyTrackPublisher.java

```diff
@@ -178,91 +179,90 @@ public void perform(@Nonnull final Run<?, ?> build,
 
         if (StringUtils.isBlank(artifact)) {
             logger.log(Messages.Builder_Artifact_Unspecified());
-            build.setResult(Result.FAILURE);
-            return;
+            throw new AbortException("Artifact not specified");
         }
         final FilePath artifactFilePath = new FilePath(filePath, artifact);
         if (projectId == null && (projectName == null || projectVersion == null)) {
             logger.log(Messages.Builder_Result_InvalidArguments());
-            build.setResult(Result.FAILURE);
-            return;
+            throw new AbortException("Invalid arguments");
         }
-        try {
```

Comment:
remove this try/catch block. Just let exceptions, including ApiClientExceptions, percolate up to the build step runner to be handled.